pydocx/openxml/markup_compatibility/fallback.py

```diff
@@ -6,10 +6,8 @@
 )
 
 from pydocx.models import XmlModel, XmlCollection
-from pydocx.openxml.wordprocessing.picture import Picture
 
 
 class Fallback(XmlModel):
     XML_TAG = 'Fallback'
-    # TODO #204: actually include all of the children defined in the spec.
-    children = XmlCollection(Picture)
+    children = XmlCollection(allow_all_children=True)
```

Comment:
I personally would spend at least an hour trying to figure out why it's not evaluating the field correctly before punting on it. The cost of merging in complicated code that may get used by others before we fix it is pretty high.